DigitalLearningSolutions.Web/Views/Login/Index.cshtml

```diff
@@ -58,9 +58,9 @@
         <span class="nhsuk-u-visually-hidden">Information: </span>
         <p>
           By logging in, I consent to my details being stored and processed in line with your
-          <a href="https://www.hee.nhs.uk/about/privacy-notice">Privacy Policy</a>
+          <a target="_blank" href="https://www.hee.nhs.uk/about/privacy-notice">Privacy Policy (opens in new window)</a>
```

Comment:
This seems a bit clunky - I wonder if we could do something proposed here: https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/

```
If your users do need to know in advance that a link goes to another website, it’s more effective to do this with words.

You can do this in the link text itself, or if you have a collection of links **you might want to do this in the immediately preceding text to avoid duplication.**
```

I don't have any good ideas though, so maybe we can send a screenshot of this into the external HEE Slack channel and ask them if they think this is ok.